src/libstd/io/stdio.rs

```diff
@@ -154,15 +154,43 @@ pub struct StdinLock<'a> {
     inner: MutexGuard<'a, BufReader<Maybe<StdinRaw>>>,
 }
 
-/// Creates a new handle to the global standard input stream of this process.
+/// Constructs a new handle to the standard input of the current process.
 ///
-/// The handle returned refers to a globally shared buffer between all threads.
-/// Access is synchronized and can be explicitly controlled with the `lock()`
-/// method.
+/// Each handle returned is a reference to a shared global buffer whose access is synchronized via
+/// a mutex. If you need more explicit control over locking, see the [lock() method][lock].
```

Comment:
Could this (and the other docs here) stick to 80 chars? I believe all of `std::io` is already formatted this way and it's nice to just keep it consistent.